src/cypress-api/index.ts

```diff
@@ -38,9 +38,9 @@ const doArchive = async ({
   await writeTestResult(
     {
       title: testTitle,
-      // doesn't matter what value we put here, as long as it's a subdirectory of where we want this to actually go
+      // this will store it at ./cypress/downloads (the last directory doesn't matter)
       // TODO: change so we don't have to do this trickery
-      outputDir: './some',
+      outputDir: './cypress/downloads/some',
```

Comment:
It's weird with this `/some` directory business in there (needed because we go [back a directory](https://github.com/chromaui/test-archiver/blob/main/src/write-archive/index.ts#L42) in `writeTestResult`), but I'd like to tackle that later when we reuse code between Playwright and Cypress.